sambacc/commands/ctdb.py

```diff
@@ -257,3 +257,10 @@ def ctdb_must_have_node(ctx: Context) -> None:
             return
         _logger.info("node not yet ready")
         waiter.wait()
+
+
+@commands.command(name="ctdb-check-ok")
+def ctdb_check_ok(ctx: Context) -> None:
+    """Check if ctdb service status is normal."""
+    _ctdb_ok()
+    ctdb.check_ok()
```

Comment:
This is not a problem with your PR, but I realized this call and the previous `_ctdb_ok()` call right after another is a tad bit confusing. However, I'm going to blame myself for choosing `_ctdb_ok` as the name for the function that checks the "feature gate"

Reply:
I was wondering if I should change the command to ctdb-status and change the corresponding function to ctdb_status(). or ctdb-nodestatus and ctdb_nodestatus()

Reply:
I think it's fine for now.